tekton/resources/nightly-release/base/kustomization.yaml

```diff
@@ -4,9 +4,8 @@ commonLabels:
 commonAnnotations:
   release: nightly
 resources:
-  - bindings.yaml
```

Comment:
Should we keep binding.yaml file here like `template.yaml` ?

Reply:
`bindings.yaml` is not in the `base` folder anymore, it's now in the root folder, because with this change bindings are not project specific anymore. I'm try to reduce the number of files to be customised when adding a new project to nightly builds.